### PR TITLE
Add test api file size, stage value not null

### PR DESCRIPTION
**Pull request by shuofan**
#### What type of this PR
feature

#### What this PR does / why we need it:
Add test api file size, custom issue stage value not null

#### Specified Reviewers:

/assign @Effet @sfwn

### CODE REVIEW

File: modules/dop/services/testcase/export.go [hunk 1]
```diff
@@ -174,8 +174,15 @@ func (svc *Service) ExportFile(record *dao.TestFileRecord) {
 			return
 		}
 
+		fileInfo, err := f.Stat()
+		if err != nil {
+			logrus.Error(apierrors.ErrExportTestCases.InternalError(err))
```
Comment (Effet) [inline modules/dop/services/testcase/export.go:179]: log should be more readable

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/dop/services/testcase/export.go (hunk 1 above)
Comment (shuofan) [reply modules/dop/services/testcase/export.go]: Done

**Review by Effet — APPROVED** [review]
